content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -140,9 +140,9 @@ The tracing setup and configuration should be run before your application code.
 
 Create a file named `tracing.ts|js`, which will contain your tracing setup code.
 
-{{< ot-tabs TypeScript JavaScript >}}
+{{< tabpane >}}
```

Comment:
Make changes in this tabpane element, similar to above that uses the header as lang.